src/TextUI/Command/Commands/CheckScopeCommand.php

```diff
@@ -0,0 +1,27 @@
+<?php declare(strict_types=1);
+/*
+ * This file is part of PHPUnit.
+ *
+ * (c) Sebastian Bergmann <sebastian@phpunit.de>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+namespace PHPUnit\TextUI\Command;
+
+use SebastianBergmann\Version;
+
+/**
+ * @internal This class is not covered by the backward compatibility promise for PHPUnit
+ */
+final class CheckScopeCommand implements Command
+{
+    public function execute(): Result
+    {
+        if (Version::class === 'PHPUnit\\SebastianBergmann\\Version') {
```

Comment:
@sebastianbergmann 

I figured it makes sense not to print the namespace of the class in case we want to verify additional class names for prefixes.

What do you think?